packages/shared/src/hooks/useSquadInvitation.ts

```diff
@@ -22,8 +24,13 @@ export const useSquadInvitation = ({
 }: UseSquadInvitationProps): UseSquadInvitation => {
   const { trackEvent } = useAnalyticsContext();
   const { completeAction } = useActions();
+  const { user } = useAuthContext();
 
-  const invitation = squad.referralUrl;
+  const invitation = addLinkShareTrackingParams(
+    squad.referralUrl,
```

Comment:
In production `squad.referralUrl` is already shortened on API side and short url returned to frontend so we either have to remove that on API side or add the tracking there already.

Reply:
good point, this is an oversight on my side. We have the same situation with invite links, which are already passed in as minified.